Makefile

```diff
@@ -12,6 +12,7 @@ pylib:
 pydep-upgrade:
 	pip install -U pip-tools
 	CUSTOM_COMPILE_COMMAND="make pydep-upgrade" pip-compile --output-file=serdio/requirements.txt serdio/pyproject.toml
+	CUSTOM_COMPILE_COMMAND="make pydep-upgrade" pip-compile --output-file=cape_encrypt/requirements.txt cape_encrypt/pyproject.toml
```

Comment:
nit: this line is unnecessary bc cape_encrypt doesn't have any dependencies to pip-compile